src/docs/development/accessibility-and-localization/internationalization.md

```diff
@@ -227,58 +310,10 @@ static MaterialLocalizations of(BuildContext context) {
 tooltip: MaterialLocalizations.of(context).backButtonTooltip,
 ```
 
-<a name="using-bundles">
-## Using the bundled Localizations&shy;Delegates
```

Comment:
@HansMuller @sfshaza2 I actually removed this section because it seems like the basic ideas here are already covered in [Setting up an internation­alized app: the flutter_localizations package](https://flutter.dev/docs/development/accessibility-and-localization/internationalization#setting-up). Let me know if you feel otherwise and we can reintroduce it.

Reply:
I'm ok with this if Hans is.